docs/iot/iot-partners/thingspark.md

```diff
@@ -0,0 +1,27 @@
+# Tizen IoT with ThingsPark Cloud
```

Comment:
Global Comment:
Is it "ThingsPark" or "thingspark"?
For more information, see https://www.thingspark.co.kr
Please verify and update throughout the document.